v3/src/components/graph/components/graph.tsx

```diff
@@ -77,18 +77,20 @@ export const Graph = observer((
 
   const toast = useToast()
 
-  const handleDropAttribute = (place: GraphPlace, attrId: string) => {
+  const handleChangeAttribute = (place: GraphPlace, attrId: string ) => {
+    if (attrId === ""){ // when request is to remove the attribute
```

Comment:
```suggestion
    if (!attrId){ // when request is to remove the attribute
```
Same reasoning as above -- simpler test and catches any nulls and undefined that might slip through as well.